tests/main_test.py

```diff
@@ -41,7 +41,7 @@ def test_scan_with_rootdir(self, mock_baseline_initialize):
             plugins=Any(tuple),
             exclude_files_regex=None,
             exclude_lines_regex=None,
-            path='test_data',
```

Comment:
I don't understand why tests won't work if I remove this. Running `detect-secrets scan test_data` seems to work properly.

Reply:
Ok! Thanks for explaining :)